src/test/utilhttp_tests.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2019 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "utilhttp.h"
+#include "test/test_bitcoin.h"
+#include <boost/test/unit_test.hpp>
+#include <iostream>
+
+BOOST_FIXTURE_TEST_SUITE(utilhttp_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(http_get_test)
+{
+    // std::cout << http_get("127.0.0.1", 8000, "/").str();
```

Comment:
how can we test?

Reply:
Yes, but do we want the unit tests to start opening sockets?

This is also indirectly tested in the elecrum_basics test in #1633

Reply:
Alright, let's do this! A custom http server implemented to test a simple http getter.